_data/settings/typography.yml

```diff
@@ -62,7 +62,7 @@ contents:
         type: px
     map:
       - key: token
-        description: Nme of the token.
+        description: Name of the token.
```

Comment:
YML is complaining about a duplicate `map` key on line `50` and `63`.

Reply:
Removed the second map because I did not feel like it added important context since they are just sample values that will be customized by the user. Let me know if you would like to show both serif and sans examples here. 

[bc77235](https://github.com/uswds/uswds-site/pull/2016/commits/bc77235cdce0a5f301d086a287d5e8272ea3a699)

Reply:
Makes sense, resolving convo.